playbooks/common/openshift-node/restart.yml

```diff
@@ -26,7 +26,7 @@
       name: openvswitch
       state: started
     when:
-    - openshift_is_containerized | bool
+    - l_is_containerized | bool
```

Comment:
Should be `openshift.common.is_containerized` so that it would be consistent with the following task